kaspresso/src/main/kotlin/com/kaspersky/kaspresso/device/screenshots/Screenshots.kt

```diff
@@ -24,4 +24,9 @@ interface Screenshots {
      * @param tag a unique tag to further identify the screenshot. Must match [a-zA-Z0-9_-]+.
      */
     fun takeAndApply(tag: String, block: File.() -> Unit)
+
+    /**
+     * @return directory used to store screenshots
+     */
+    fun getScreenshotDir(): File
```

Comment:
For what purpose is it added? First, I see the only usage in a sample test, second, it semantically doesn't belong to this interface.

Reply:
Ok, thanks, I'll fix that